opencog/cython/opencog/atomspace.pxd

```diff
@@ -124,6 +124,10 @@ cdef extern from "opencog/atoms/base/Atom.h" namespace "opencog":
         string get_name()
         vector[cHandle] getOutgoingSet()
 
+        bool operator=(cAtom&)
```

Comment:
Am I right that `operator==` should be used instead?
```suggestion
        bool operator==(cAtom&)
```

Reply:
yes, thanks